bbb_view.php

```diff
@@ -129,6 +129,17 @@
         bigbluebuttonbn_event_log(\mod_bigbluebuttonbn\event\events::$events['meeting_left'], $bigbluebuttonbn);
         // Update the cache.
         $meetinginfo = bigbluebuttonbn_get_meeting_info($bbbsession['meetingid'], BIGBLUEBUTTONBN_UPDATE_CACHE);
+        // Check the origin page.
+        $sql = "SELECT meta FROM {bigbluebuttonbn_logs}
+                  where userid = ? AND (log = 'Join' OR log = 'Create')
```

Comment:
You don't have to look for the Create event. It is the Join the only that matters for this effect.

Reply:
Right. I added a new commit.